pkg/apis/sealed-secrets/v1alpha1/sealedsecret_expansion.go

```diff
@@ -285,12 +285,16 @@ func (s *SealedSecret) Unseal(codecs runtimeserializer.CodecFactory, privKeys ma
 
 		for key, value := range s.Spec.Template.Data {
 			var plaintext bytes.Buffer
-			template, err := template.New(key).Parse(value)
+			t := template.New(key)
+			if (s.Spec.Template.Delimeters.Left != "") && (s.Spec.Template.Delimeters.Right != "") {
```

Comment:
If somebody passes this config:

```yaml
spec:
  template:
    delimeters:
      right: "<"
```

then the setting will be silently ignored, which is confusing.

The go template docs says:

> An empty delimiter stands for the corresponding default: {{ or }}

So I think it's safe to just call `t.Delims(s.Spec.Template.Delimeters.Left, s.Spec.Template.Delimeters.Right)` without any conditional

Reply:
Yeah, I agree. I'll try it just after I figure out how to force `make generate` to work :smile:

Reply:
oh I forgot about 

```
bash: ../../../../vendor/k8s.io/code-generator/generate-groups.sh: No such file or directory
```